test/libscap/test_suites/userspace/scap_ppm_sc.cpp

```diff
@@ -100,8 +100,8 @@ TEST(scap_ppm_sc, scap_get_events_from_ppm_sc)
 			struct syscall_evt_pair pair = g_syscall_table[sys_id];
 			if(pair.ppm_sc == ppm_sc)
 			{
-				ASSERT_TRUE(events_array[pair.enter_event_type]) << "ppm_sc: " << scap_get_syscall_info_table()[pair.ppm_sc].name << " (" << pair.ppm_sc << ") should be associated with event: " << pair.enter_event_type << std::endl;
-				ASSERT_TRUE(events_array[pair.exit_event_type]) << "ppm_sc: " << scap_get_syscall_info_table()[pair.ppm_sc].name << " (" << pair.ppm_sc << ") should be associated with event: " << pair.exit_event_type << std::endl;
+				ASSERT_TRUE(events_array[pair.enter_event_type]) << "ppm_sc: " << scap_get_ppm_sc_name((ppm_sc_code)pair.ppm_sc) << " (" << pair.ppm_sc << ") should be associated with event: " << pair.enter_event_type << std::endl;
```

Comment:
@FedeDP and @Andreagit97 plz confirm, this was the broken test we didn't notice.

Btw still unsure why the `generic` subfolder just wasn't activated, not like it makes a lot of sense. I quickly just unified all of them into `userspace`, but open to refactor to something else while ensuring each test suite is loaded.

@Andreagit97 only test that was not ported over to test/libscap was the recently added common_strl, it's now ported over.

Reply:
Yep for some reason they were skipped :/ Great catch!